### PR TITLE
Manual Entry: Fix Decimal Place Entry in BMP Efficiencies

### PR DESCRIPTION
**Pull request by rajadain**
## Overview

Previously the decimal place limitation was only applied to field with labels. This excluded the fields in the BMP Efficiencies modal, which because of their tight arrangement do not have labels. This was a mistake, since they need to follow the same decimal place behavior as others.

Connects #2995 

### Demo

image

## Testing Instructions

* Check out this branch and `bundle`
* Ensure each manual entry form follows the appropriate decimal place limitations:
  - Land Cover: 1
  - Animals: 0
  - Everything Else: 3

### CODE REVIEW

File: src/mmw/js/src/modeling/gwlfe/entry/views.js [hunk 1]
```diff
@@ -277,6 +277,28 @@ var FieldView = Marionette.ItemView.extend({
...
+        }
+
+        var autoValue = this.model.get('autoValue'),
+            decimalPlaces = this.model.get('decimalPlaces'),
+            // How many decimal places are allowed for the field in the UI
+            //   decimalPlaces = 0 => step = 1
+            //   decimalPlaces = 1 => step = 0.1
+            //   decimalPlaces = 3 => step = 0.001
+            // and so on.
+            step = Math.pow(10, -decimalPlaces);
```
Comment (kellyi) [inline src/mmw/js/src/modeling/gwlfe/entry/views.js]: :+1:

**Comment by kellyi** [comment]
Testing this out now!

**Comment by kellyi** [comment]
Fix is working well! However, I noticed one additional set of fields that seem like they have to have integer values in order for the model to run: the "Number of Persons..." fields in the "Waste Water" tab:

screen shot 2018-10-15 at 12 55 33 pm

Assuming the fix is now pretty trivial to do, seems like it makes sense to fix that here too.

**Comment by rajadain** [comment]
Done in 349318c

image

**Comment by kellyi** [comment]
screen shot 2018-10-15 at 1 54 23 pm

**Comment by rajadain** [comment]
Thanks for taking a look!

**Pull request merged**